edx_solutions_api_integration/courses/views.py

```diff
@@ -1772,6 +1772,12 @@ def get(self, request, course_id):  # pylint: disable=W0613
 
 class CoursesMetricsGradesLeadersList(SecureListAPIView):
     """
+    **DEPRECATION WARNING**
```

Comment:
The view you linked is the same as this one, isn't it? I'll "un-deprecate" the the other one though and wait for the confirmation on this one, if I misunderstood which one you are referring to.

Reply:
@BogdanL Sorry, not sure what I was looking at. This one is correct, the deprecation warning should stay.

Reply:
@BogdanL @mtyaka 

I'm still confused about this deprecation:

* This one returns "top 3 users" so the new API is not a direct replacement
* This API is used for server-to-server API calls to power the Apros LMS, and it doesn't make sense to me that we'd want Apros to be using an API in the "mobile" namespace.

So I don't think this deprecation warning makes sense.

We do eventually want the Apros LMS to use an OAuth2 API to retrieve the leaders instead of an API-key based view, but the team at edX and Arbisoft are already working on that transition.

Reply:
@bradenmacdonald In the ticket description it says "There is an API for that which already exists and is used by Apros (obviously), but there are two problems with the [existing API endpoints](https://github.com/edx-solutions/api-integration/blob/9d0f1a603e935f00bfd1b935e1738c663e4bf165/edx_solutions_api_integration/courses/views.py#L1722-L1790)" which made me think this was the main view to deprecate, although it's serving different purposes than the one we introduce here. 

But one of the acceptance criteria is to "The existing proficiency APIs in the api-integration are marked as deprecated but will continue to work.", so I'm wondering which ones should we deprecate if not the ones (with the `user_id` in the URL) that Apros is already using?

Reply:
@BogdanL Ah, I see. Let's remove that from the acceptance criteria then. If the APIs don't serve the exact same purpose, we shouldn't mark the old one as deprecated.

Reply:
That makes sense. I updated the JIRA issue description to reflect this decision.